CHANGELOG.md

```diff
@@ -16,6 +16,11 @@
 
 ---
 
+## v4.7.1
+
+- 🐛 Adjust records generated during view change so their date matches the view date ([#1486](https://github.com/DataDog/browser-sdk/pull/1486))
+- 🐛 Fix log has already been spied ([#1450](https://github.com/DataDog/browser-sdk/pull/1450))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏this is not really user facing since it is a unit test issue

Reply:
Erf yes